BotRoyaleUnity/Assets/Scripts/Networking/SocketConnectionHandler.cs

```diff
@@ -126,10 +128,19 @@ public void EmitGameMessage(JSONObject data, Action<string> onMessageSent = null
         }
         else
         {
-            socket.Emit("game-message", data.ToString(), onMessageSent); ;
+            socket.Emit("game-message", data.ToString(), onMessageSent);
         }
     }
 
+    public void EmitEmptyParts(string name){
```

Comment:
This method seems to not be used anywhere, I believe you moved it to another class, so it can be deleted